[FLINK-25686][Connector/Pulsar] Support schema evolution for Pulsar source and sink. (#12)
apache/flink-connector-pulsar: docs/content/docs/connectors/datastream/pulsar.md

you can use the predefined `PulsarDeserializationSchema`. Pulsar connector provides three implementation methods.

- Decode the message by using Pulsar's [Schema](https://pulsar.apache.org/docs/en/schema-understand/).

>   If using KeyValue type or Struct types, the pulsar `Schema` does not contain type class info which
>   is needed by `PulsarSchemaTypeInformation`. So we provide two more APIs to pass the type info.

Reviewer: ```suggestion
  If using KeyValue type or Struct types, the pulsar `Schema` does not contain type class info. But it is
  still needed to construct `PulsarSchemaTypeInformation`. So we provide two more APIs to pass the type info.
```
